### PR TITLE
✨ Tina Live Editing Docs

### PR DESCRIPTION
**Pull request by joshbermanssw**
This covers ✨ Apply Live Editing to Tina Docs  #1947  🔨 Fix table of Contents Queries and wire correctly https://github.com/tinacms/tina.io/issues/1961  and Remove all old HTML from md pages https://github.com/tinacms/tina.io/issues/1960. 

What we have done after making all .md docs pages to .mdx pages 
- [ ] applied the `useTina()` hook in the docs `[...slug].tsx`, and wired this data up to using Client gql query in getStaticProps instead of legacy `getDocProps` method 
- [ ] Reformatted this data to fill in the previous and next buttons correctly with the `<DocsPagination>` component
- [ ] Changed the previous and next from a `string` input to a `reference` input 
- [ ] Created a new helper method `getSeoDescription` which takes in the doc body and extracts upto 160 words from it, this is because the legacy `getDocProps` just passed through a variable `excerpt` before and now we have gotten rid of it. 
- [ ] Created a new helper method `getTableOfContents` which takes in the AST body and extracts all `h2` and `h3` items into an array so we can use the existing ToCListener
- [ ] Removed all raw html that existed on the .md pages previously that now do not render on the .mdx pages. 
- [ ] Replaced this raw HTML by adding the following templates to the .mdx pages { Youtube Embed, Warning Callout, Iframe embed, Cloudinary Video, Image and Text (see figure below), MinimizeText Tab (see video below) 

I created the Image and Text tab so that we could have images nexto code blocks. However, there is currently an issue with saving a rich text field inside of another rich text field, so whilst the component works for normal text, it is currently unavailable for code blocks. So as a work around for now we have put all code blocks above images and created an issue to come back and fix when the bug is resolved in the TinaCMS backlog (https://github.com/tinacms/tina.io/issues/1967) 

Screenshot 2024-07-31 at 9 34 32 AM

**Figure: 📝 Example of what ImageAndText looks like**

I created the `MinimizeText` tab to create sections that minimize and open upon clicking (see video below). However this faces the same issue as spoken about above, TinaCMS currently it fails to save a rich text field within another rich text field 

https://github.com/user-attachments/assets/702ec365-2fb4-459e-9098-d3b0748472e8

**Figure: 📹 Example of MinimizeText field**

### CODE REVIEW

File: content/docs/forestry/migrate.mdx [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
 
 ## Introduction
 
 TinaCMS can be added to your site locally. In this doc, we'll guide you through the local setup, as well as editing on your production site.
 
 <div style="position:relative;padding-top:56.25%;">
-  <iframe width="560" frameborder="0" allowfullscreen
-      style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/OHY8PyeLK1g" title="Migrating from TinaCMS to Forestry" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share">
-  </iframe>
+  <iframe
```
Comment (bradystroud) [inline content/docs/forestry/migrate.mdx:10]: This should be replaced with youtube embed component

File: content/docs/editing/blocks.mdx [hunk 2]
```diff
@@ -160,7 +161,10 @@ query PagesDocument($relativePath: String!) {
...
+<Iframe
+  iframeSrc="/api/graphiql/?query=%7B%0A%20%20pages(relativePath%3A%20%22turbo.json%22)%20%7B%0A%20%20%20%20blocks%20%7B%0A%20%20%20%20%20%20__typename%0A%20%20%20%20%20%20...%20on%20PagesBlocksHero%20%7B%0A%20%20%20%20%20%20%20%20__typename%0A%20%20%20%20%20%20%20%20tagline%0A%20%20%20%20%20%20%20%20headline%0A%20%20%20%20%20%20%20%20text%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20...%20on%20PagesBlocksFeatures%20%7B%0A%20%20%20%20%20%20%20%20__typename%0A%20%20%20%20%20%20%20%20items%20%7B%0A%20%20%20%20%20%20%20%20%20%20title%0A%20%20%20%20%20%20%20%20%20%20text%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20...%20on%20PagesBlocksContent%20%7B%0A%20%20%20%20%20%20%20%20__typename%0A%20%20%20%20%20%20%20%20body%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A"
+  height={400}
+/>
```
Comment (bradystroud) [inline content/docs/editing/blocks.mdx:167]: This should be a GraphiQl component, not Iframe.

File: content/docs/graphql/queries/add-document.mdx [hunk 3]
```diff
@@ -1,9 +1,8 @@
 ---
 id: /docs/graphql/queries/add-document
 //: null
```
Comment (bradystroud) [inline content/docs/graphql/queries/add-document.mdx:3]: This look like a bug

File: content/docs/graphql/queries/add-document.mdx [hunk 4]
```diff
@@ -12,6 +11,9 @@ The `addPendingDocument` mutation will take in the collection name as a paramete
 
 > Note: `addPendingDocument` does not currently support fields of any kind, just creating the record.
 
-<iframe loading="lazy" src="/api/graphiql/?query=mutation%20%7B%0A%20%20addPendingDocument(collection%3A%20%22post%22%2C%20relativePath%3A%20%22pedro.json%22)%20%7B%0A%20%20%20%20__typename%0A%20%20%7D%0A%7D&operationName=GetBlogPost" width="800" height="400" />
+<Iframe
+  iframeSrc="/api/graphiql/?query=mutation%20%7B%0A%20%20addPendingDocument(collection%3A%20%22post%22%2C%20relativePath%3A%20%22pedro.json%22)%20%7B%0A%20%20%20%20__typename%0A%20%20%7D%0A%7D&operationName=GetBlogPost"
+  height={400}
+/>
```
Comment (bradystroud) [inline content/docs/graphql/queries/add-document.mdx:17]: same as other comment - should be a GraphiQL component.

File: content/docs/graphql/queries/advanced/filter-documents.mdx [hunk 5]
```diff
@@ -35,28 +34,43 @@ Conditions can be either "binary" or "ternary". A binary condition is composed o
 
 Here we will query our `post` collection with `postConnection` and filter the results by the post `title`:
 
-<iframe width="800" height="400" loading="lazy" src="/api/graphiql/?query=%7B%0A%20%20postConnection(filter%3A%20%7Btitle%3A%20%7BstartsWith%3A%20%22Vote%22%7D%7D)%20%7B%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20title%0A%20%20%20%20%20%20%20%20category%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A" />
+<Iframe
+  iframeSrc="/api/graphiql/?query=%7B%0A%20%20postConnection(filter%3A%20%7Btitle%3A%20%7BstartsWith%3A%20%22Vote%22%7D%7D)%20%7B%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20title%0A%20%20%20%20%20%20%20%20category%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A&#x22; "
+  height={400}
+/>
```
Comment (bradystroud) [inline content/docs/graphql/queries/advanced/filter-documents.mdx:40]: same as other comment - should be a GraphiQL component.

File: content/docs/graphql/queries/advanced/filter-documents.mdx [hunk 6]
```diff
@@ -35,28 +34,43 @@ Conditions can be either "binary" or "ternary". A binary condition is composed o
...
 
 ### Filtering on a field with the IN operator
 
 Here we will query our `post` collection with `postConnection` and filter the results so that only members of the specified `category` are returned:
 
-<iframe width="800" height="400" loading="lazy" src="/api/graphiql/?query=%7B%0A%20%20postConnection(filter%3A%20%7Bcategory%3A%20%7Bin%3A%20%5B%22politics%22%5D%7D%7D)%20%7B%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20title%0A%20%20%20%20%20%20%20%20category%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A" />
+<Iframe
+  iframeSrc="/api/graphiql/?query=%7B%0A%20%20postConnection(filter%3A%20%7Bcategory%3A%20%7Bin%3A%20%5B%22politics%22%5D%7D%7D)%20%7B%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20title%0A%20%20%20%20%20%20%20%20category%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A&#x22; "
+  height={400}
+/>
```
Comment (bradystroud) [inline content/docs/graphql/queries/advanced/filter-documents.mdx:49]: same as other comment - should be a GraphiQL component.

File: content/docs/graphql/queries/advanced/filter-documents.mdx [hunk 7]
```diff
@@ -35,28 +34,43 @@ Conditions can be either "binary" or "ternary". A binary condition is composed o
...
 
 ### Filtering on a date range
 
 Here we will query our `post` collection with `postConnection` and filter the results so that only posts with a `date` between the specified range are returned:
 
-<iframe width="800" height="400" loading="lazy" src="/api/graphiql/?query=%7B%0A%20%20postConnection(filter%3A%20%7Bdate%3A%20%7Bafter%3A%20%222022-06-01T07%3A00%3A00.000Z%22%2C%20before%3A%20%222022-06-30T07%3A00%3A00.000Z%22%7D%7D)%20%7B%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20title%0A%20%20%20%20%20%20%20%20category%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A" />
+<Iframe
+  iframeSrc="/api/graphiql/?query=%7B%0A%20%20postConnection(filter%3A%20%7Bdate%3A%20%7Bafter%3A%20%222022-06-01T07%3A00%3A00.000Z%22%2C%20before%3A%20%222022-06-30T07%3A00%3A00.000Z%22%7D%7D)%20%7B%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20title%0A%20%20%20%20%20%20%20%20category%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A&#x22; "
+  height={400}
+/>
```
Comment (bradystroud) [inline content/docs/graphql/queries/advanced/filter-documents.mdx:58]: same as other comment - should be a GraphiQL component.

File: content/docs/graphql/queries/advanced/filter-documents.mdx [hunk 8]
```diff
@@ -35,28 +34,43 @@ Conditions can be either "binary" or "ternary". A binary condition is composed o
...
 
 ### Filtering on multiple fields
 
 It is possible to filter on multiple fields. Multiple conditions are currently treated as a boolean `AND` operation. Here we will query our `post` collection with `postConnection` and filter the results by `category` and `title`:
 
-<iframe width="800" height="400" loading="lazy" src="/api/graphiql/?query=%7B%0A%20%20postConnection(filter%3A%20%7Bcategory%3A%20%7Bin%3A%20%5B%22politics%22%5D%7D%2C%20title%3A%20%7BstartsWith%3A%20%22Vote%22%7D%7D)%20%7B%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20title%0A%20%20%20%20%20%20%20%20category%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A" />
+<Iframe
+  iframeSrc="/api/graphiql/?query=%7B%0A%20%20postConnection(filter%3A%20%7Bcategory%3A%20%7Bin%3A%20%5B%22politics%22%5D%7D%2C%20title%3A%20%7BstartsWith%3A%20%22Vote%22%7D%7D)%20%7B%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20title%0A%20%20%20%20%20%20%20%20category%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A&#x22; "
+  height={400}
+/>
```
Comment (bradystroud) [inline content/docs/graphql/queries/advanced/filter-documents.mdx:67]: same as other comment - should be a GraphiQL component.

File: content/docs/graphql/queries/advanced/filter-documents.mdx [hunk 9]
```diff
@@ -35,28 +34,43 @@ Conditions can be either "binary" or "ternary". A binary condition is composed o
...
 
 #### Filtering on a reference
 
 Here we will query our `post` collection with `postConnection`, and filtering on the referenced `author`'s name:
 
-<iframe width="800" height="400" loading="lazy" src="/api/graphiql/?query=%7B%0A%20%20postConnection(filter%3A%20%7Bauthor%3A%20%7Bauthor%3A%20%7Bname%3A%20%7Beq%3A%20%22Napolean%22%7D%7D%7D%7D)%20%7B%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20title%0A%20%20%20%20%20%20%20%20category%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A" />
+<Iframe
+  iframeSrc="/api/graphiql/?query=%7B%0A%20%20postConnection(filter%3A%20%7Bauthor%3A%20%7Bauthor%3A%20%7Bname%3A%20%7Beq%3A%20%22Napolean%22%7D%7D%7D%7D)%20%7B%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20title%0A%20%20%20%20%20%20%20%20category%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A&#x22; "
+  height={400}
+/>
```
Comment (bradystroud) [inline content/docs/graphql/queries/advanced/filter-documents.mdx:76]: same as other comment - should be a GraphiQL component.

File: content/docs/graphql/queries/advanced/pagination.mdx [hunk 10]
```diff
@@ -32,7 +31,10 @@ To page through query results in the forward direction, the `first` and `after`
...
 
-<iframe width="800" height="500" loading="lazy" src="/api/graphiql/?query=%7B%0A%20%20postConnection(sort%3A%20%22date%22%2C%20first%3A%201%2C%20after%3A%20%22cG9zdCNkYXRlIzE2NTUyNzY0MDAwMDAjY29udGVudC9wb3N0cy92b3RlRm9yUGVkcm8uanNvbg%3D%3D%22)%20%7B%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20title%0A%20%20%20%20%20%20%20%20date%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%20%20pageInfo%20%7B%0A%20%20%20%20%20%20hasNextPage%0A%20%20%20%20%20%20endCursor%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A" />
+<Iframe
+  iframeSrc="/api/graphiql/?query=%7B%0A%20%20postConnection(sort%3A%20%22date%22%2C%20first%3A%201%2C%20after%3A%20%22cG9zdCNkYXRlIzE2NTUyNzY0MDAwMDAjY29udGVudC9wb3N0cy92b3RlRm9yUGVkcm8uanNvbg%3D%3D%22)%20%7B%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20title%0A%20%20%20%20%20%20%20%20date%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%20%20pageInfo%20%7B%0A%20%20%20%20%20%20hasNextPage%0A%20%20%20%20%20%20endCursor%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A&#x22; "
+  height={400}
+/>
```
Comment (bradystroud) [inline content/docs/graphql/queries/advanced/pagination.mdx:37]: same as other comment - should be a GraphiQL component.

File: content/docs/reference/toolkit/fields/blocks.mdx [hunk 11]
```diff
@@ -1,19 +1,30 @@
...
+    stroke="currentColor"
+    fill="currentColor"
+    stroke-width="0"
+    viewBox="0 0 512 512"
+    height="1em"
+    width="1em"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path d="M32 464h448L256 48 32 464zm248-64h-48v-48h48v48zm0-80h-48v-96h48v96z"></path>
+  </svg>
```
Comment (bradystroud) [inline content/docs/reference/toolkit/fields/blocks.mdx:27]: SHould be replaced with a component

File: content/docs/reference/toolkit/fields/color.mdx [hunk 12]
```diff
@@ -10,12 +9,24 @@ consumes:
...
+    stroke="currentColor"
+    fill="currentColor"
+    stroke-width="0"
+    viewBox="0 0 512 512"
+    height="1em"
+    width="1em"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path d="M32 464h448L256 48 32 464zm248-64h-48v-48h48v48zm0-80h-48v-96h48v96z"></path>
+  </svg>
```
Comment (bradystroud) [inline content/docs/reference/toolkit/fields/color.mdx:29]: SHould be replaced with a component

File: content/docs/reference/types.mdx [hunk 13]
```diff
@@ -18,3 +17,8 @@ Tina contains multiple tools and applications designed to improve collaboration
 - [`reference`](/docs/reference/types/reference/)
 - [`object`](/docs/reference/types/object/)
 - [`rich-text`](/docs/reference/types/rich-text/)
+
+<MinimizeText
+  heading="Here we go!"
+  text={<>Test me please Now a second line</>}
+/>
```
Comment (bradystroud) [inline content/docs/reference/types.mdx:24]: I dont like the name, should be based off 
https://www.w3schools.com/tags/tag_summary.asp

File: content/toc-doc.json
Comment (bradystroud) [inline content/toc-doc.json:1]: Make sure there are redirects for all these changed urls

File: pages/docs/[...slug].tsx [hunk 14]
```diff
@@ -107,7 +115,17 @@ export const getStaticProps: GetStaticProps = async function (props) {
...
+    const tinaDocsNavigation = await client.queries.getAllDocs()
+    //TOOD: get rid of line 121. https://github.com/tinacms/tina.io/issues/1982
+    const oldNavDocs = await getDocsNav()
+    return {
+      props: {
+        new: { results },
+        allDocs: { tinaDocsNavigation },
+        oldNavDocs,
+      },
+    }
```
Comment (bradystroud) [inline pages/docs/[...slug].tsx:128]: Lets get rid of all the old stuff, we dont need it.
If something is using it, change it to use the new stuff

File: tina/schema.tsx [hunk 15]
```diff
@@ -124,20 +124,113 @@ export const schema = defineSchema({
           },
         },
         {
-          name: 'prev',
-          label: 'Prev',
-          type: 'string',
+          name: 'Next',
+          label: 'Next',
+          type: 'reference',
+          collections: ['doc', 'examples'],
```
Comment (bradystroud) [inline tina/schema.tsx:130]: Lowercase name for consistency
```suggestion
          name: 'next',
          label: 'Next page',
          type: 'reference',
          collections: ['doc', 'examples'],
```

File: tina/schema.tsx [hunk 16]
```diff
@@ -124,20 +124,113 @@ export const schema = defineSchema({
...
         },
         {
-          name: 'next',
-          label: 'Next',
-          type: 'string',
+          name: 'previous',
+          label: 'Previous',
```
Comment (bradystroud) [inline tina/schema.tsx:134]:
```suggestion
          label: 'Previous page',
```

File: tina/schema.tsx [hunk 17]
```diff
@@ -124,20 +124,113 @@ export const schema = defineSchema({
...
+              name: 'Youtube',
+              label: 'Youtube Embed',
+              fields: [
+                {
+                  type: 'string',
+                  name: 'embedSrc',
+                  label: 'Embed URL',
+                  description:
+                    'Looks like this https://www.youtube.com/embed/Yoh2c5RUTiY',
+                },
```
Comment (bradystroud) [inline tina/schema.tsx:154]: I am not sure if this works with other styles of youtube URLs 
e.g. `https://www.youtube.com/watch?v=l0e9i8zXcIs`

The description should be clearer on what input the code can handle
```suggestion
                {
                  type: 'string',
                  name: 'embedSrc',
                  label: 'Embed URL',
                  description:
                    '⚠️ Only youtube embed urls work - they look like this https://www.youtube.com/embed/Yoh2c5RUTiY',
                },
```

File: utils/docs/getDocIds.ts
Comment (bradystroud) [inline utils/docs/getDocIds.ts:1]: Not clear what this does - add a comment

File: utils/docs/getTableOfContens.ts
Comment (bradystroud) [inline utils/docs/getTableOfContens.ts:1]: typo in the filename

Right now there is a ToC on the left and a ToC on the right... I think we need to rename one of them or the code will be very confusing

**Review by bradystroud — CHANGES_REQUESTED** [review]
Hey @joshbermanssw 

HUGE PR
Left plenty of feedback, I think its on the right track though

File: content/docs/data-fetching/custom-queries.mdx [hunk 18]
```diff
@@ -13,18 +13,16 @@ const myPost = await client.queries.post({ relativePath: 'HelloWorld.md' })
...
 ## Extending the auto-generated `client.queries`
 
 The client can be extended to perform more advanced queries (including querying multiple root collections at once), by adding queries to the `tina/queries` directory. All files that end in `gql` or `graphql` in this directory will be added to the client.
 
 Tina generates custom [GraphQL fragments](https://graphql.org/learn/queries/#fragments) to be used in custom queries so that the queries will not have to be updated when the schema is updated.
 
-<!-- TODO: add internal docs link -->
-
-> Note: These fragments can be seen in [`tina/__generated__/frags.gql`]()
+> Note: These fragments can be seen in `tina/__generated__/frags.gql`()
```
[the remark this answers is not in the document]
Comment (joshbermanssw) [reply content/docs/data-fetching/custom-queries.mdx:25]: ✅

File: content/docs/forestry/migrate.mdx (hunk 1 above)
Comment (joshbermanssw) [reply content/docs/forestry/migrate.mdx:10]: ✅

File: content/docs/graphql/queries/add-document.mdx (hunk 3 above)
Comment (joshbermanssw) [reply content/docs/graphql/queries/add-document.mdx:3]: resolved✅

File: utils/docs/getTableOfContens.ts
Comment (joshbermanssw) [reply utils/docs/getTableOfContens.ts:1]: ToC on left is not using any words using ToC now 👍 Typo fixed 👍

File: utils/docs/getDocIds.ts
Comment (joshbermanssw) [reply utils/docs/getDocIds.ts:1]: ✅

File: tina/schema.tsx (hunk 17 above)
Comment (joshbermanssw) [reply tina/schema.tsx:154]: ✅

File: tina/schema.tsx (hunk 16 above)
Comment (joshbermanssw) [reply tina/schema.tsx:134]: ✅

File: tina/schema.tsx (hunk 15 above)
Comment (joshbermanssw) [reply tina/schema.tsx]: ✅

File: content/docs/editing/blocks.mdx (hunk 2 above)
Comment (joshbermanssw) [reply content/docs/editing/blocks.mdx:167]: As per our conversation, i disagree

File: pages/docs/[...slug].tsx (hunk 14 above)
Comment (bradystroud) [reply pages/docs/[...slug].tsx:128]: Read https://github.com/tinacms/tina.io/issues/1982
duh

**Review by bradystroud — APPROVED** [review]
Approved - tested pretty much every page in the docs and it seems to work

**Pull request merged**